lib/actions/config.js

```diff
@@ -18,6 +18,13 @@ import * as commandOptions from './command-options';
 
 import { normalizeUuidProp } from '../utils/normalization';
 import { getBalenaSdk, getVisuals } from '../utils/lazy';
+import * as _ from 'lodash';
```

Comment:
lodash is used for top level command creation so it will always be easily loaded, and it's generally used in so many places that there won't be a benefit to lazy loading it until a lot more of its usage is removed